Resources/config/services.yml

```diff
@@ -1,42 +1,49 @@
 services:
+    _defaults:
+        bind:
+            $userClass: '%imatic_user.entity.user.class%'
+            $groupClass: '%imatic_user.entity.group.class%'
+            $resettingFromEmail: '%imatic.user.email.resseting_from%'
+
     # Form
     imatic_user.form.type.profile:
         class: Imatic\Bundle\UserBundle\Form\Type\User\ProfileType
-        arguments: ["%fos_user.model.user.class%"]
+        autowire: true
```

Comment:
bundly obecne by se nemely spolehat na autowire a autoconfigure, byl bych pro dodrzovani best practices https://symfony.com/doc/current/bundles/best_practices.html#services

Reply:
@ldfk pokud vím tak best practices nedodržujeme. V té samé sekci se píše:

> If the bundle defines services, they must be prefixed with the bundle alias instead of using fully qualified class names like you do in your project services.

přesto jsi u data bundlu přejmenoval services podle class names: https://github.com/imatic/data-bundle/blob/master/UPGRADE-4.0.md#services ? Nebo dodržujeme jen něco?

Reply:
@nenadalm zbytecne me chytas za slovo. o pojmenovani services sem se prece vubec nezminoval. jde mi o `autowire` a `autoconfigure` a o to ze v tom bych nasledoval best practices. dalsi vysvetleni proc je treba zde https://symfony.com/doc/current/service_container/autowiring.html#public-and-reusable-bundles. a dalsi duvod, je ten bundle je verejny (neni pouzivany jen v ramci imaticu) a nemuzes spolehat na to, ze bude vzdy pouzit na projektu, ktery ma povolene `autowire` a `autoconfigure`.

Reply:
> @nenadalm zbytecne me chytas za slovo.

nikoliv. Tvůj argument byl že se to má dělat podle best practices, tak jsem se jen logicky divil tomu že ty best practices také nedodržuješ, takže ten argument je trochu divný.

> nemuzes spolehat na to, ze bude vzdy pouzit na projektu, ktery ma povolene autowire a autoconfigure

o tom že autowire/autoconfigure jde nějak zakázat jsem nikdy neslyšel a teď se mi to také nedaří najít. Můžeš poslat nějaký odkaz na dokumentaci?

Reply:
@stepankoci 
@nenadalm ma chut se hadat, ja o to nestojim a nemam na to cas. muzes prosim rozhodnout (jako treti osoba)? Vznesl sem navrh ze by se nemelo v services pouzivat `autowire` a `autoconfigure` na zaklade dokumetace [tady](https://symfony.com/doc/current/bundles/best_practices.html#services) a [tady](https://symfony.com/doc/current/service_container/autowiring.html#public-and-reusable-bundles), Mila si to nemysli. Rozhodni tedy prosim jak to bude, diky.

Reply:
@ldfk @nenadalm prosím udělejme to jak píše Lukáš, souhlasím s ním.

Reply:
opraveno.